blockchain/src/blockchain/accounts.rs

```diff
@@ -37,7 +37,9 @@ impl Blockchain {
 
                 // Check if the receipts contain an error.
                 if let Err(e) = batch_info {
-                    return Err(PushError::AccountsError(e));
+                    // TODO: This is due to a dependency between blockchain and accounts that is not wasm friendly
+                    log::error!(" Received accounts error while commiting accounts: {}", e);
```

Comment:
Extra space and typo:
```suggestion
                    log::error!("Received accounts error while committing accounts: {}", e);
```